packages/store-sync/src/common.ts

```diff
@@ -76,7 +76,12 @@ export type StorageAdapterBlock = { blockNumber: BlockLogs["blockNumber"]; logs:
 export type StorageAdapter = (block: StorageAdapterBlock) => Promise<void>;
 
 // TODO: adjust when we get namespace support (https://github.com/latticexyz/mud/issues/994) and when table has namespace key (https://github.com/latticexyz/mud/issues/1201)
-export const schemasTable = storeConfig.tables.Tables;
+// TODO: adjust when schemas are automatically resolved
+export const schemasTable = {
+  ...storeConfig.tables.Tables,
+  valueSchema: resolveUserTypes(storeConfig.tables.Tables.valueSchema, storeConfig.userTypes),
+};
+
```

Comment:
kind of annoying, but we need this workaround for now to land the contract changes (until we have a better way to resolve the user type config with strong types)

Reply:
hopefully coming in #1561